### PR TITLE
disable dark theme, remove all Scene related stuff, make new director…

### PR DESCRIPTION
**Pull request by virustyt**
…ies for files and reorder files

### CODE REVIEW

File: FinanceAdvisor/Support/AppDelegate.swift [hunk 1]
```diff
@@ -11,15 +11,15 @@ import CoreData
...
+        
+        let window = UIWindow(frame: UIScreen.main.bounds)
+        window.makeKeyAndVisible()
+        
         return true
     }
 
     // MARK: UISceneSession Lifecycle
-
     func application(_ application: UIApplication, configurationForConnecting connectingSceneSession: UISceneSession, options: UIScene.ConnectionOptions) -> UISceneConfiguration {
```
Comment (pandafliesinspace) [inline FinanceAdvisor/Support/AppDelegate.swift:23]: Scene related methods also can be deleted

**Comment by pandafliesinspace** [comment]
Also please disable iPad support and leave only portrait device orientation

File: FinanceAdvisor/Support/AppDelegate.swift [hunk 2]
```diff
@@ -11,15 +11,15 @@ import CoreData
 @main
 class AppDelegate: UIResponder, UIApplicationDelegate {
```
Comment (pandafliesinspace) [inline FinanceAdvisor/Support/AppDelegate.swift:13]: Add `var window: UIWindow?` here. It will be needed in future to assign the root viewcontroller

**Review by pandafliesinspace — APPROVED** [review]
Can be merged after fixes

File: FinanceAdvisor/Support/AppDelegate.swift (hunk 2 above)
Comment (virustyt) [reply FinanceAdvisor/Support/AppDelegate.swift:13]: done

File: FinanceAdvisor/Support/AppDelegate.swift (hunk 1 above)
Comment (virustyt) [reply FinanceAdvisor/Support/AppDelegate.swift]: done

**Comment by virustyt** [comment]
Disable iPad support and leaved only portrait device orientation in project. Also make two other fixes, wich are desribed in messages below

**Review by pandafliesinspace — APPROVED** [review]
lgtm

**Comment by pandafliesinspace** [comment]
@virustyt please merge this PR

**Pull request merged**